hash/c64_cass.xml

```diff
@@ -153,6 +337,76 @@ Some are based on the C64 Tape Archive at https://archive.org/download/c64tapes/
 		</part>
 	</software>
 
+	<software name="5thgearp" cloneof="5thgear">
+		<description>5th Gear (Prism)</description>
+		<year>1992</year>
+		<publisher>Prism Leisure</publisher>
+
+		<part name="cass1" interface="cbm_cass">
+			<dataarea name="cass" size="557418">
+				<rom name="5th_Gear.tap (prism)" size="557418" crc="dd8af099" sha1="795512a880f8812799f4baa674eac1ea6fae52d7"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="6hits">
+		<description>6 Computer Hits</description>
+		<year>1985</year>
+		<publisher>Beau-Jolly</publisher>
+
+		<part name="cass1" interface="cbm_cass">
+			<feature name="part_id" value="Cadcam Warrior / Battle Through Time / Heroes of Karn / Eddie Kidds Jump Challenge / Rocket Roger / Cave Fighter"/>
+			<dataarea name="cass" size="2968544">
+				<rom name="6_Computer_Hits.tap" size="2968544" crc="7f7cfcef" sha1="1f3686bf2b421f22479410cece958aa9ee5c34e7"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="6hitsa" cloneof="6hits">
+		<description>6 Computer Hits (alt)</description>
+		<year>1985</year>
+		<publisher>Beau-Jolly</publisher>
+
+		<part name="cass1" interface="cbm_cass">
+			<feature name="part_id" value="Cadcam Warrior / Battle Through Time / Heroes of Karn / Eddie Kidds Jump Challenge / Rocket Roger / Cave Fighter"/>
+			<dataarea name="cass" size="2968543">
+				<rom name="6_Computer_Hits_a1.tap" size="2968543" crc="72399639" sha1="91369bd3314a36f0424dd364fa59f3bd252c0b4b"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="6pak">
+		<description>6 Pak</description>
+		<year>1987</year>
+		<publisher>Elite Systems</publisher>
+
+		<part name="cass1" interface="cbm_cass">
+			<feature name="part_id" value="Side A: Scooby Doo / 1942 / Duet"/>
+			<dataarea name="cass" size="1826748">
+				<rom name="6_Pak_Side_1.tap" size="1826748" crc="e213fb21" sha1="b7a6ef8d7d27467ce6dc24839e43a8e1a0ab8250"/>
+			</dataarea>
+		</part>
+
+		<part name="cass2" interface="cbm_cass">
+			<feature name="part_id" value="Side B: Jet Set Willy II / Sacred Armour of Antiriad / Fighting Warrior / Split Personalities"/>
+			<dataarea name="cass" size="2419415">
+				<rom name="6_Pak_Side_2.tap" size="2419415" crc="0ba93379" sha1="a3ae97aa38974642ee391002351882d1aeaa72c3"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="720">
+		<description>720</description>
```

Comment:
Is this actually a game called “720” or is this a variant of the skateboarding game “720°” for C64?

Reply:
It's the US Gold conversion of 720° most likely

Reply:
It is the US Gold conversion of 720 Degrees.  Description probably needs to be updated to show 720 Degrees in the same manner as the arcade version in MAME to be consistent.